storm.go

```diff
@@ -18,7 +18,9 @@ func Open(path string, stormOptions ...func(*DB) error) (*DB, error) {
 	}
 
 	for _, option := range stormOptions {
-		option(s)
+		if err = option(s); err != nil {
```

Comment:
I know that all options `func` in `options.go` return nil, at this point. but it's better to validate if later any options return a non `nil`